### PR TITLE
Release 3.20.0

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

**Review by babastienne — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 3.20.0 (2024-05-21)
+
+**🚀 New features**
+
+- Add a `displaySecondaryLayersByDefault` settings to display or not secondary layers by default (#997 and #1130)
```
Comment (camillemonchicourt) [inline docs/changelog.md:7]:
```suggestion
- Add a `displaySecondaryLayersByDefault` settings (default to `true`) to display or not secondary layers on maps by default (#997 and #1130)
```

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+- Display Files from attachments to donwload in details pages (#1118)
```
Comment (camillemonchicourt) [inline docs/changelog.md:8]:
```suggestion
- Display Files from attachments to download in details pages (#1118)
```

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+
+**✨ Improvements**
+
+- Clean search params in search page (#1100)
+- Use Image component for TrekMarker map Popup and DetailsPreview logo images (#1102)
+- Improve detailsCard UI interaction (#1112)
+- Rename attachments props to images props (#1113)
+- Update homepage BannerCarousel autoplaySpeed to 5sec for images (#863 and #1120)
+- Improve mobile filters menu (#1006 and #1131)
+- Break words for large links from description details (#1133)
```
Comment (camillemonchicourt) [inline docs/changelog.md:18]:
```suggestion
- Break words for large links from description details (#1114 and #1133)
```

**Review by camillemonchicourt — COMMENTED** [review]
A few suggestions in Changelog.

**Comment by dtrucs** [comment]
> A few suggestions in Changelog.

Thx for the review :+1:

**Pull request merged**